yogstation/code/modules/mob/living/simple_animal/hostile/floor_cluwne.dm

```diff
@@ -366,6 +367,7 @@ GLOBAL_VAR_INIT(floor_cluwnes, 0)
 
 
 /mob/living/simple_animal/hostile/floor_cluwne/proc/Kill(mob/living/carbon/human/H)
+	if(H)
```

Comment:
🤔 this if() doesn't do anything?

Reply:
it should check for when it pulls someone down if its a human which should prevent the bug of the floor cluwne bugging out I think

Reply:
wait that didnt work one moment